docs/scos/dev/front-end-development/202212.0/oryx/building-components/create-component.md

```diff
@@ -0,0 +1,211 @@
+
+The above pattern is commonly used in all Oryx domains. It ensure efficiently consumption of backend APIs and efficient rendering of DOM elements.
+
+### Step 3: Configure the component
+
+Oryx components can be made configurable by options. [Component options](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-components/component-options.html) can be provided statically to the application or load from a backend API. Component options are an important concept in Oryx as they allow components to be reusable cross different business models. For example, a component can render different results based on an option that is provided `true` for a b2c application, but `false` for a b2b application.
+
+The component options are resolved by the `ContentMixin`, similar to how the `ProductService` resolves the product data. It is possible to combine multiple mixins in your component implementation, for example:
+
+```ts
+import { resolve } from "@spryker-oryx/di";
+import { ContentMixin } from "@spryker-oryx/experience";
+import { ProductMixin } from "@spryker-oryx/product";
+
+interface ProductIdOptions {
+  myOption?: boolean;
+}
+
+export class ProductIdComponent extends ProductMixin(
+  ContentMixin<ProductIdOptions>(LitElement)
+) {
+  protected override render(): TemplateResult {
+    const { myOption } = this.$options();
+
+    if (!myOption) return;
+
+    return html`id: ${this.$product()?.id}`;
+  }
+}
+```
+
+You can provide default options in the component as well as provide options in feature sets or in the application, see [Component options](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-components/component-options.html).
+
+### Step 4: Style the component DOM
+
+Oryx components are styled with standard CSS. The components have a separate DOM attached, also known as the [shadow DOM](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM), which means that the component styles are processed in isolation. Consequently, component styles will not _leak_ into other components, nor do any global styles cascade down to the component.
+
+Styling components in the shadow DOM might be new experience to you. This guide is not going to provide detailed knowledge on this topic, as it's a standard feature. However, there are few things to know when it comes to Oryx and styling components:
+
+- Oryx uses design system components, that are provided in the UI package. For example, `<oryx-button>`, `<oryx-link>`, etc. are components that you can use to ensure a common visual language. Moreover, design system components can be customized, similar to any Oryx component.
+- Font styles rules like `font-face` or `font-size` will, unlike other CSS rules,cascade to web components, no matter how deep they are nested. Oryx provides standard font rules in the `<oryx-app>` component.
```

Comment:
Probably we will need a link where described what is `<oryx-app>` component.

Reply:
yes that would nicely fit under building applications section.